controller/Controller.php

```diff
@@ -52,7 +52,10 @@ function __construct($f3, $db)
     function tutorsPage($param)
     {
         //checking to see if user is logged in. If not logged in, will redirect to login page
-        $this->isLoggedIn();
+        $this->isLoggedIn($_SESSION['user_id']);
+        //if non admin tires to access tutorsPage it redirects them to their log in page
+        if ($this->_db->checkAdmin($_SESSION['user_id'])['user_is_admin'] == 0)
```

Comment:
Use {} when doing if statements even if they're just one line. That's been our standard and it's more readable.